projects/core/src/occ/adapters/product/default-occ-product-config.ts

```diff
@@ -12,6 +12,7 @@ export const defaultOccProductConfig: OccConfig = {
           details:
             'products/${productCode}?fields=averageRating,stock(DEFAULT),description,availableForPickup,code,url,price(DEFAULT),numberOfReviews,manufacturer,categories(FULL),priceRange,multidimensional,tags,images(FULL)',
           attributes: 'products/${productCode}?fields=classifications',
+          price: 'products/${productCode}?fields=price(formattedValue)',
```

Comment:
Does the fact that the price is ini 2 scopes affect the way we want it to refresh?

Reply:
Having taken a look at the solution, one question comes to mind.  the price was already in 2 of thee existing scopes.  Could we have fixed the issue without creating a new scope that contains just the price?  What is the added value of creating a new scope in this case?